lib/ui/widgets/post_content/index.dart

```diff
@@ -10,9 +12,10 @@ import 'widgets/export.dart';
 class PostContent extends StatelessWidget {
   final Post post;
   const PostContent({Key key, @required this.post}) : super(key: key);
-
   @override
   Widget build(BuildContext context) {
+    List<String> shouldShowLinkPreview = _isLinkInContent();
```

Comment:
I think it's better to rename this variable accordingly to its later usage. Something like `previewUris` would be better IMO. What do you think?